### DISCUSSION TITLE
database navigator

### DISCUSSION THREAD

**Discussion by ct-groups** [dbeaver/dbeaver#14679] [General]
i can't expend all in database navigator,how to do that

**Comment by LonwoLonwo** [comment]
Hello @rainman-code 
If I understand you correctly, DBeaver doesn't have such a type of option. It can be very dangerous - expand all folders in the Database Navigator at one time.

All other helpful buttons are here:
![2021-11-25 10_21_34-DBeaver 21 3 0 - _localhost_ Script-33](https://user-images.githubusercontent.com/45152336/143397336-50c6056f-b81e-42a9-92cb-1e447e47b1d8.png)

**Reply by ct-groups** [reply]
I find a way that gives a shortcut key to expanding selected,then I use the shortcut key to implement it
